lib/Helper/Install.php

```diff
@@ -414,6 +414,11 @@ public function step7(Request $request, Response $response) : Response
             $library_location = $library_location . '/';
         }
 
+        // Attempt to create fonts sub-folder in Library location
+        if (!mkdir($library_location . 'fonts', 0777, true)) {
```

Comment:
This is a great addition here, but I think maybe `ensureLibraryExists()` should have it too?

Reply:
OK cool, as you are then!